magento/magento2
# Free Currency Converter API requires API key now.

### Description (*)

Free Currency Converter API now requires API key, this change is to add the API key setting to backend and append the key to request url.

### Fixed Issues (if relevant)

magento/magento2#22468

### Manual testing scenarios (*)

1. Navigate to Stores -> Configuration -> General -> Currency Setup
2. Make sure the new API Key field is added for Currency Converter API.
3. Navigate to Stores -> Currency Rates
4. Select Currency Converter API as Import Service and click Import.
5. Make sure the rated value(s) are refreshed.

### Contribution checklist (*)
